dedibox-network/rpn/reference-content/private-networks-overview.mdx

```diff
@@ -15,7 +15,7 @@ categories:
 When deploying infrastructure on the Internet, network connectivity will play a crucial part in your success in terms of both performance and security. You want your services to run quickly, smoothly and with minimum latency, while also ensuring that sensitive data is fully protected even as it is being transferred over a public network like the Internet.  
 ​
 
-The Internet was initially designed to enable a few academics and researchers to share data between themselves. As such, security was not a paramount concern. Of course, over time the Internet grew, with more and more private companies and personal users taking it up, until today it has become the omnipresent global network we all know, linking billions of machines and people worldwide. While technologies have evolved and data-protection mechanisms to secure our connections have become more sophisticated, we should never forget that the Internet remains essentially a public network. This means that infrastructure attacks like Distributed Denial-of-Service or Bruteforce can always stand to pose a threat to your infrastructure and your data.  
+The Internet was initially designed to enable a few academics and researchers to share data between themselves. As such, security was not a paramount concern. Of course, over time the Internet grew, with more and more private companies and personal users taking it up, until today it has become the omnipresent global network we all know, linking billions of machines and people worldwide. While technologies have evolved and data-protection mechanisms to secure our connections have become more sophisticated, we should never forget that the Internet remains essentially a public network. This means that infrastructure attacks like Distributed Denial-of-Service or Brute force can always stand to pose a threat to your infrastructure and your data.  
 ​
 
 Scaleway provides a range of solutions to help you protect against such attacks and secure your infrastructure. Besides DDoS protection, we also offer several ways to create private networks, fully compliant with [RFC 1918](https://docs.umbrella.com/umbrella-user-guide/docs/internal-networks-setup-guide). These networks can add extra protection for your data, since they are entirely isolated, and completely separate from the Internet or any public network.
```

Comment:
```suggestion
Scaleway provides a range of solutions to help protect you against such attacks and secure your infrastructure. Besides DDoS protection, we also offer several ways to create private networks, fully compliant with [RFC 1918](https://docs.umbrella.com/umbrella-user-guide/docs/internal-networks-setup-guide). These networks can add extra protection for your data, since they are entirely isolated, and completely separate from the Internet or any public network.
```